src/main.rs

```diff
@@ -1,3 +1,7 @@
+#[macro_use]
```

Comment:
add the list of macro you are importing in please, `table` ?

```suggestion
#[macro_use(table)]
```

Reply:
Will do.

Reply:
Couldn't do it. It complains about missing stuff, some diesel black magic.